src/ex4/client/src/components/itemComp/__tests__/ItemCompSnapshot.test.jsx

```diff
@@ -0,0 +1,23 @@
+import { create, act } from "react-test-renderer";
+import ItemComp from "../ItemComp";
+
+const items = [
+	{
+		id: 56,
+		itemName: "Take dog out for a walk",
+		status: false,
+	},
+	{
+		id: 32,
+		itemName: "Do the dishes",
+		status: true,
+	},
+];
+
+test("renders currectly item, item must include id,itemName,status", () => {
+	let root;
+	act(() => {
+		root = create(items.map((item) => <ItemComp key={item.id} item={item} />));
```

Comment:
since you are checking the item component here, it would be better to check each set of attributes in it's own test. instead of having an array of items, have 2 tests.